the-incredible-bulk/src/main/java/gov/va/api/health/bulkfhir/service/controller/publication/DefaultPublicationStatusTransformer.java

```diff
@@ -17,6 +19,7 @@ public PublicationStatus apply(Iterable<StatusEntity> statusEntities) {
     log.info("{}", statusEntities);
     Counter counter = new Counter();
     PublicationStatusBuilder publication = PublicationStatus.builder();
+    List<FileStatus> fileStatuses = new ArrayList<>();
```

Comment:
could this be `new ArrayList<>(statusEntities.size())` ?

Reply:
Good catch